pkg/consts/metric.go

```diff
@@ -231,3 +231,29 @@ const (
 	MetricsMemFilePerNumaCgroup  = "mem.file.numa.cgroup"
 	MetricsMemAnonPerNumaCgroup  = "mem.anon.numa.cgroup"
 )
+
+const (
```

Comment:
this defines metric rather than metrics (to emit); actually, we should the metrics in wherever requires them, and no need to define them in a general place

Reply:
They are defined here because they are referenced by eviction and memory advisor.

Reply:
they should not be defined here, the reason is as above

Reply:
generally, we don't encourage different components to share the same metric names

Reply:
maybe we should use different name for these two "metrics", it‘s really easy to get confused. For example:
runtime metric: the information about running containers, pods and system;
metric: the data we used to monitor katalyst itself.

for this certain case, I agree with @waynepeking348 . Maybe someday we will deploy advisor and eviction manager seperately and they might be maintained by different people. If they use same metrics name, it will be another confusing thing because it's not only reflect the status about one component.